lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/api/VortexAggregateFuture.java

```diff
@@ -182,6 +180,8 @@ public void run() {
       });
     }
 
+    LOG.log(Level.SEVERE, "PUT IN RESULT QUEUE!");
```

Comment:
If this is not just for debugging, could you make the message informative, with more proper logging level?